### ISSUE TITLE
Ben's feedback: Online shopping vs prices in the souk?

### ISSUE THREAD

**Issue by arnoldyb** [w201rdada/portfolio-pdurkin84#1]
The gist of your idea is that more foreign investment would come into the Middle East if investors could have more trust in one of the key economic indicators -- the Consumer Price Index. You propose creating a CPI that’s independent of government tampering by scraping online shopping websites for prices. 

In general, I understand the problem well from your writeup. But I recommend reorganizing the presentation of the problem to make it clear right from the beginning who needs the data and who would profit from providing it. In fact, something like the paragraph above where I summarized the gist of your project might be the sort of intro that would work. 

I don’t doubt that you are right about the idea that “investors may feel suspicious of the national data and as a result not be comfortable investing, or if they do decide to invest, they could require higher returns to compensate for higher risk.” But I would see if you could establish this is true with third-party research before going too far in the project, since it lies at the crux of the demand for what you are proposing to build. 

I really liked this sentence where you acknowledge that the region is not monolithic: “Given the multi-cultural makeup of some these countries and the vast disparity between incomes classes, it will be prudent to create multiple baskets reflecting typical buying patterns within the groups.” I would go a bit further addressing this. First: Why would you build a CPI for the Middle East as opposed to Middle Eastern countries? What goes on in the Egyptian economy bears no resemblance to what goes on in Bahrain’s. Second: Isn’t online shopping still an activity mainly for the rich, and don’t prices online reflect that? For a CPI, don’t you need to know what ordinary people are paying for onions in the souk -- or does it not matter if the prices are inflated as long as you track changes in the same inflated price over time? You might actually be able to get the souk prices by tapping into market data used by farmers and wholesalers, or by finding some creative way to encourage people across the region to crowdsource it.  

In conclusion, I like that you swung big, taking on a problem where the solution could move billions of dollars. This feels like an idea that could be very usefully applied to China where there is serious doubt in the government numbers. I spent a little bit of time in the world of selling information about emerging markets, and I am guessing you know that whatever number you publish would quickly circulate and lose all direct sales value. Presumably, your client would actually be an investment bank that has an interest in driving more investment in the region.
